Gems/EMotionFX/Code/Source/Editor/ColliderHelpers.cpp

```diff
@@ -191,7 +204,82 @@ namespace EMotionFX
         return false;
     }
 
-    void ColliderHelpers::AddCopyFromMenu(QObject* parent, QMenu* parentMenu, PhysicsSetup::ColliderConfigType createForType, const QModelIndexList& modelIndices,
+    void ColliderHelpers::AddToRagdoll(const QModelIndexList& modelIndices)
+    {
+        if (modelIndices.empty())
+        {
+            return;
+        }
+
+        const AZStd::string groupName = AZStd::string::format("Add joint%s to ragdoll", modelIndices.size() > 1 ? "s" : "");
+
+        MCore::CommandGroup commandGroup(groupName);
+
+        AZStd::vector<AZStd::string> jointNames;
```

Comment:
I think you could get away with just using string references.

Reply:
We (me and @ablepharus) chatted about this offline and agreed this is likely okay to keep as-is to avoid any kind of lifetime issues. If this does become a performance issue we might be able to switch to `string_view`